classes/class-wc-connect-nux.php

```diff
@@ -342,6 +342,7 @@ public function set_up_nux_notices() {
 
 			switch ( $banner_to_display ) {
 				case 'before_jetpack_connection':
+				case 'tos_only_banner':
```

Comment:
Old case can be removed: https://github.com/Automattic/woocommerce-services/blob/85dbbc637e4ba68da659bce11999a125f3807e0b/classes/class-wc-connect-nux.php#L372-L375

Reply:
Yep, I missed that. I'm going to rewrite this commit to include the changes. Force pull after.

Reply:
Fixed.